.vitepress/config.mts

```diff
@@ -8,48 +8,127 @@ import ViteUnoCSS from 'unocss/vite'
 import { mermaid } from './md-mermaid'
 import { katex } from '@mdit/plugin-katex'
 
-function getGuideSidebar(): DefaultTheme.SidebarItem[] {
+function nav(): DefaultTheme.NavItem[] {
   return [
     {
-      text: 'Getting started',
+      text: 'Guide',
       items: [
+        // { text: 'Get Started', link: '/guide/get-started/index' },
+        // { text: 'Build and Install', link: '/guide/get-started/install' },
+        // { text: 'Tutorials', link: '/guide/get-started/tutorials'},
         {
-          text: 'How Iroha works',
-          link: '/',
+          text: 'Get Started',
+          items: [
+            { text: 'Build and Install', link: '/guide/get-started/install' },
+            { text: 'SDK Tutorials', link: '/guide/get-started/tutorials' },
+          ],
```

Comment:
It feels confusing that the first item in "Getting Started" navbar section doesn't lead to the first sidebar item in "Getting Started" section ("About Iroha").

"SDK Tutorials" is also not under "Get Started" sidebar section, it is a separate one.

Not critical, just feels inconsistent.